README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.org/T-Systems-MMS/puppet-secc_sshd.svg?branch=master)](https://travis-ci.org/T-Systems-MMS/puppet-secc_sshd)
 
-####Table of Contents
+## Table of Contents
 
 1. [Overview](#overview)
```

Comment:
TOC needs update

Reply:
missing content removed, please check again
TOC is working!